CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ CHECK_INCLUDE_FILES ("gdb/jit-reader.h" HAVE_GDB_JIT_READER_H)
 option(BUILD_TESTS "Build unit tests to ensure sanity" TRUE)
 option(BUILD_FEX_LINUX_TESTS "Build FEXLinuxTests, requires x86 compiler" FALSE)
 option(BUILD_THUNKS "Build thunks" FALSE)
-option(BUILD_FEXCONFIG "Build FEXConfig, requires SDL2 and X11" TRUE)
+set(USE_FEXCONFIG_TOOLKIT "imgui" CACHE STRING "If set, build FEXConfig (qt or imgui)")
```

Comment:
Do we want to make `qt` the default before merging it?

Reply:
We'll need to update our CI machines before it can be switched to default. Setting the cmake option until then should be fine.

Reply:
Also probably doesn't hurt to ship this as opt-in for one release, just in case edge case bugs come up :)